src/plugins/intel_gpu/src/graph/eltwise.cpp

```diff
@@ -111,70 +114,37 @@ std::vector<layout> eltwise_inst::calc_output_layouts(eltwise_node const& /*node
     auto out_data_type = desc->output_data_type.value_or(input_layout.data_type);
 
     auto get_output_layout = [&]() {
-        const auto& autob = desc->broadcast_spec;
         auto out_pshape = input_layout.get<ShapeType>();
         cldnn::format out_format = input_layout.format;
 
+        // We create dummy Add op as shape infer is exactly the same for any eltwise op type, so there is no need to have correct op type
+        ov::op::v1::Add op;
+        op.set_autob(desc->broadcast_spec);
+
+        std::vector<ShapeType> output_shapes = {ShapeType()};
+        std::vector<ShapeType> input_shapes;
+        for (size_t i = 0; i < desc->input_size(); i++) {
+            input_shapes.push_back(impl_param.get_input_layout(i).get<ShapeType>());
+        }
+        eltwise_shape_infer(&op, input_shapes, output_shapes);
+
```

Comment:
If I remember correctly, we need legacy eltwise broadcast (broadcast every dimension to max) for prepare_quantize. @ahnyoung-paul  Is there no problem with removing the legacy shape infer?

Reply:
I checked with int8 model and there was no problem.

Reply:
I've checked previous failed tests (cldnn unit test and gpu functional tests) for legacy shape infer and also checked several Bert models. I think it is no problem.

Reply:
@ahnyoung-paul "Previous failing tests" <= I guess these are not using new shape infer, are they? In my memory the issue was on the prepare_quantize. Anyway it seems okay because I checked the i8 bert model.

Reply:
@yeonbok, I reproduced that smoke_LPT/ConvolutionQDqTransformation.CompareWithRefImpl/f16* test is failed when allow_new_shape_infer is set to true in gpu functional test because of that the elimination of legacy eltwise broadcast for prepare_quantize code. I also it seems okay because bert models are working well and all current tests are passed. however, I think we need to update it for the next steps.